index.d.ts

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+
+export interface BusyIndicatorProperties {
```

Comment:
Could we call this `BusyIndicatorProps`, so it matches the react naming convention

Reply:
Sure !